lib/perf_hooks.js

```diff
@@ -72,10 +72,9 @@ const kQueued = Symbol('queued');
 const kTimerified = Symbol('timerified');
 const kInsertEntry = Symbol('insert-entry');
 const kGetEntries = Symbol('get-entries');
-const kIndex = Symbol('index');
-const kMarks = Symbol('marks');
 const kCount = Symbol('count');
 
+const performanceSet = new Set();
```

Comment:
`performanceSet` as a global variable will not work, it should be in the class attribute.

Reply:
I wanted to refactor it as a private field just now, but `make doc-only` failed.

and `private field` not supported in acorn without extending `acorn-class-fields`

https://github.com/nodejs/node/blob/17f323ebfaf4b6f15e994412113d1e856a2e0ffc/tools/doc/apilinks.js#L57-L59

another way is to move this to `internal` but it uses many other variables

Reply:
and, why not work? class Performance only use once and not be exported

Reply:
Sorry, I didn't notice that class `Performance` was not exported and it still works. But it is better to set it as a `private field`

Reply:
yes, sure. but as I said above, there will throw an [error](https://github.com/nodejs/node/pull/32652#issuecomment-608998721)

Reply:
I would definitely prefer not to move this out of the class.